lib/fastlane/plugin/wpmreleasetoolkit/actions/common/buildkite_trigger_build_action.rb

```diff
@@ -0,0 +1,90 @@
+module Fastlane
+  module Actions
+    class BuildkiteTriggerBuildAction < Action
+      def self.run(params)
+        require 'buildkit'
+
+        UI.message "Triggering build on branch #{params[:branch]} at #{params[:commit]}"
+
+        pipeline_name = {
+          PIPELINE: params[:pipeline_file]
+        }
+
+        client = Buildkit.new(token: params[:buildkite_token])
+        response = client.create_build(
+          params[:buildkite_organization],
+          params[:buildkite_pipeline],
+          {
+            branch: params[:branch],
+            commit: params[:commit],
+            env: params[:environment].merge(pipeline_name)
+          }
+        )
+
+        response.state == 'scheduled' ? UI.message('Done!') : UI.user_error!("Failed to start job\nError: [#{response}]")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Triggers a job on Buildkite'
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(
+            key: :buildkite_token,
+            env_name: 'BUILDKITE_ACCESS_TOKEN',
+            description: 'Buildkite Personal Access Token',
+            type: String
+          ),
+          FastlaneCore::ConfigItem.new(
+            key: :buildkite_organization,
+            env_name: 'BUILDKITE_ORGANIZTION',
+            description: 'The Buildkite organization that contains your pipeline',
+            type: String,
+            default_value: 'wordpress-mobile'
```

Comment:
I'd like to suggest removing the default value. 

I'm saying this because, in the long run, the wordpress-mobile org won't hold the majority of usages for this action. At that point, it might be confusing to have such a default. If we don't set it now, we also save ourselves the trouble of removing it in the future (which would be a breaking change, no?) and keep all usages more verbose but consistent.

```suggestion
            type: String
```

Reply:
+1 on removing the default value here.

Reply:
Yep – was back and forth on this because [the current one works this way](https://github.com/wordpress-mobile/release-toolkit/blob/b533fe2376454931c3196dddcfb672b3c1e329c8/lib/fastlane/plugin/wpmreleasetoolkit/actions/common/circleci_trigger_job_action.rb#L33-L37).

FWIW this is for the Buildkite account, not for the GitHub org, but still, this kinda pushes me over the edge to say "let's manually specify it".

Reply:
> FWIW this is for the Buildkite account, not for the GitHub org, but still, this kinda pushes me over the edge to say "let's manually specify it".

Oh, right 🤦 

That changes things a bit, actually 🤔 I'm not so strongly for removing it. But yeah, an extra line of verbosity won't hurt.